inc/search.class.php

```diff
@@ -971,8 +1099,10 @@ static function constructSQL(array &$data) {
     * @param  boolean $is_having Do we construct sql WHERE or HAVING part
     *
     * @return string             the sql sub string
+    *
+    * @since 10.0.0 Method is no longer static
     */
-   static function constructCriteriaSQL($criteria = [], $data = [], $searchopt = [], $is_having = false) {
+   public function constructCriteriaSQL($criteria = [], $data = [], $searchopt = [], $is_having = false) {
```

Comment:
1. Should be mentionned in changelog.

2. This method is only privately used, may be changed to private.